site/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta.adoc

```diff
@@ -126,13 +126,60 @@ Return to the Configure SAML Tab.  This can be found by navigating as follows fr
+* Optionally set the value of [field]#Default RelayState# if you want to provide a specific redirect URI. If this value is omitted, the first Authorized redirect URI found in the FusionAuth Application OAuth configuration will be used. This option is supported in FusionAuth version greater than or equal to `1.41.0`. In prior versions, append `redirect_uri=` to the value specified [field]#Single sign on URL# to control the final redirect URI.  For more information link:/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta#optional-adding-a-specific-redirect-after-saml-login[see section below].
 * Set the [field]#Application username# to be `Email`.
 
 image::identity-providers/idp-initiated/okta-sign-on-advanced-settings.png[Configure Okta with the FusionAuth SP information,width=1200]
+There are a few optional workflows (adding a persistent Id, sending over an email attribute, and configuring a custom redirect) you can employ listed below in the next *two* sections.  If these are not needed, please save the Okta application by clicking [breadcrumb]#"Next"# until complete and skip down.
 
-Save the Okta application by clicking [breadcrumb]#"Next"# until complete.
+=== Optional: Adding a Specific Redirect After SAML login
 
+*Scenario:* you want a user to return to a specific redirect url after SAML auth is complete, there are two options to specify this url:
+
+1. Add to Relay State field
+- As of version `1.41.0` FusionAuth allows adding a redirect in the relay state.
+- Simply add something like `https://hooli.com` to the [field]#Default RelayState# in your Okta configuration.
+2. Add to the ACS
+- You can append your redirect to the [field]#Single sign-on URL# (ACS)
+- Example: +
+`<host>/samlv2/acs/22cea679-83af-422a-806a-baae792b3ab9/85a03867-dccf-4882-adde-1a79aeec50df?redirect_uri=http://hooli.com`
+
+image::identity-providers/idp-initiated/saml-acs-and-relay-state.png[Showing Setting the Relay and ACS values,width=1200, role=bottom-cropped]
+
+The above values will need to be added to the link:/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta#configure-the-fusionauth-application-redirect-url[Authorized redirect URLs] OAuth configuration of your application in FusionAuth. Additional information can be found link:/docs/v1/tech/identity-providers/samlv2-idp-initiated/#saml-v2-integration-details[in our SAML. overview documentation].
+
+=== Optional: Setting a Persistent Id and Sending Email as an Attribute
+
+Ensure you are still on the Configure SAML Tab.  This can be found by navigating as follows from the App homepage: [breadcrumb]#General > SAML Settings > Edit Button > General Settings > Next Button > Configure SAML# tab in the Okta Admin screen.
```

Comment:
We typically use one space after a period. For some reason there are two spaces here.